packages/worker/src/api/controllers/global/users.ts

```diff
@@ -300,6 +301,10 @@ export const find = async (ctx: any) => {
 
 export const tenantUserLookup = async (ctx: any) => {
   const id = ctx.params.id
+  // is email, check its valid
+  if (id.includes("@") && !emailValidator.validate(id)) {
```

Comment:
I don't think this will work because the ID could be a user id `us_...` I believe,